requirements.txt

```diff
@@ -1,7 +1,30 @@
-coverage
-disclosure-extractor==0.0.52
-docker
-eyed3
-pytest
+Django==3.2.6
+gunicorn==20.1.0
+PyPDF2
+Pillow
 requests
+pdf2image
+boto3>=1.9.38
```

Comment:
This one, `flask`, and a few others might be possible to be removed?

Reply:
And my plan was to switch to poetry.